docs/source/background/nodes.md

```diff
@@ -0,0 +1,60 @@
+## Nodes
+
+Nodes are coroutine-based asynchronous relational operators that perform operations on data.  Messages (data buffer) is read in and written to via [channels](./channels.md)
+
+
+**Python**
+
+```python
+async def accumulator(ctx: Context, ch_out: Channel, ch_in: Channel):
+    """Sum Column"""
+
+    total = 0
+    while (msg := await ch_in is not None:)
+        chunk = TableChunk.from_message(msg)
+        total = SUM(chunk.column)
+    await send(total)
+```
+
+**C++**
```

Comment:
These code snippets don't really work, I think. If you want to show a pattern I think it is better with pseudo-code. Or else have a complete example.